src/main/java/it/gov/pagopa/pu/sil/endpoint/PuForOrganizationPaymentsEndpoint.java

```diff
@@ -26,9 +27,25 @@ public PaaSILAutorizzaImportFlussoRisposta paaSILAutorizzaImportFlusso(
       @SoapHeader("{http://www.regione.veneto.it/pagamenti/ente/ppthead}intestazionePPT") SoapHeaderElement header){
     IntestazionePPT intestazionePPT = SoapUtils.unmarshallHeader(header, IntestazionePPT.class);
     log.info("processing paaSILAutorizzaImportFlusso codIpaEnte[{}]", intestazionePPT.getCodIpaEnte());
+    PaaSILAutorizzaImportFlussoRisposta response = new PaaSILAutorizzaImportFlussoRisposta();
+
+    //check if the logged user has the right to call this endpoint
+    if(!SecurityUtils.isValidWsSilUser(intestazionePPT.getCodIpaEnte())){
+      log.error("User [{}] not authorized to call paaSILAutorizzaImportFlusso for organization {}",
+        SecurityUtils.getLoggedUser().getUserId() ,intestazionePPT.getCodIpaEnte());
+      return FaultUtils.setFaultOnResponse(
```

Comment:
could be useful to have a setNotAuthorizedFaultOnResponse in order to centralize the fault code and description?